assets/chezmoi.io/docs/developer-guide/contributing-changes.md

```diff
@@ -35,17 +35,25 @@ that:
 * The code passes [`golangci-lint`][golangci-lint]. You can ensure this by
   running `make lint`.
 
-* The commit messages follow the [conventional commits specification][commits].
-  chezmoi's release notes are generated directly from the commit messages. The
-  following criteria can be used to determine the commit type:
+* The commit messages adhere to the [conventional commits specification][commits],
+  with the following additional requirements:
+
+      * The first character of the commit message is uppercase, e.g:
+
+          `chore: Fix typo in test`
+
+      * The commits not scoped.
+
+    chezmoi's release notes are generated directly from the commit messages. The
+    following criteria can be used to determine the commit type:
 
       * Small changes, such as fixing a typo or correcting grammar: `chore`
       * Bug fixes: `fix`
       * Extending an existing feature, or adding a new feature: `feat`
       * Adding to, or updating, the documentation: `docs`
       * Anything not covered by the above: `chore`
```

Comment:
I would consider rewording this as well:

```markdown
     * `fix`: bug fixes in chezmoi code
     * `feat`: extending an existing feature or adding a new feature
     * `docs`: adding to or updating the documentation
     * `chore`: small changes, such as fixing a typo, correcting grammar (including in documentation), or anything not covered by the above
```

Reply:
I agree that this is better because all of the types are aligned.